python/ray/air/tests/test_remote_storage_hdfs.py

```diff
@@ -0,0 +1,30 @@
+"""Test remote_storage in a ci environment with real hdfs setup."""
+import os
+
+import pytest
+
+from ray.air._internal.remote_storage import get_fs_and_path
+
+
+def test_get_fs_and_path_hdfs():
+    # the following file is written in `install-hdfs.sh`.
+    with open("/tmp/hdfs_env", "r") as f:
+        for line in f.readlines():
+            line = line.rstrip("\n")
+            tokens = line.split("=")  # assuming that vars don't have "="
+            os.environ[tokens[0]] = tokens[1]
+    import sys
+
+    sys.path.insert(0, os.path.join(os.environ["HADOOP_HOME"], "bin"))
+    hostname = os.getenv("CONTAINER_ID")
```

Comment:
Let's move this whole block into a `pytest.fixture` that yields the hdfs_hostname